platform/android/MapboxGLAndroidSDKTestApp/src/test/java/com/mapbox/mapboxsdk/geometry/LatLngBoundsTest.java

```diff
@@ -34,47 +35,40 @@ public void beforeTest() {
       .build();
   }
 
-  @Test
-  public void testSanity() {
```

Comment:
A sanity test becomes obsolete when there other tests that indirectly test the same thing (this is a common test driven development practice).